src/components/exercises/ExercisesList.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC, useState, useEffect } from 'react';
+
+import { Button } from 'antd';
+
+import { Card, Loading } from '../custom';
+
+import * as api from '../../api';
+
+import { Exercise } from '../../types';
+
+import ExerciseItem from './ExerciseItem';
+
+import './ExercisesList.css';
+
+const ExercisesList: FC<{
+    history: any
+}> = ({ history }) => {
+  // recup exercises
```

Comment:
if your code is in english, then your comments should definitely be too

Reply:
also, this does not really add to the code

Reply:
when I read this new file, I have a feeling that the comments you said were sort of missing on the project are more some documentation on the project's architecture than inline comments in the code. Am I correct?

Reply:
Yes you're right, I deleted this comment.